api/hasura/actions/_handlers/deleteUserBulk.ts

```diff
@@ -0,0 +1,142 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { DateTime } from 'luxon';
+
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { errorResponseWithStatusCode } from '../../../../api-lib/HttpError';
+import { authUserDeleterBulkMiddleware } from '../../../../api-lib/userDeleterBulk';
+import {
+  deleteUserBulkInput,
+  composeHasuraActionRequestBody,
+} from '../../../../src/lib/zod';
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  const {
+    input: { payload },
+    session_variables: sessionVariables,
+  } = composeHasuraActionRequestBody(deleteUserBulkInput).parse(req.body);
+
+  const { circle_id, addresses } = payload;
+  const existingUserIds : number[] = [];
+  const uniqueAddresses = [...new Set(addresses.map(a => a.toLowerCase()))];
+
+  // Check whether circle is in active epoch
+  const {
+    epochs: [currentEpoch],
```

Comment:
think this query is also no longer needed anymore

Reply:
The circle doesn't need to be in an active epoch in order to delete the users?

Reply:
nope, you can delete users anytime